content/en/docs/concepts/security/secrets-good-practices.md

```diff
@@ -62,12 +62,16 @@ recommendations include:
 *  Implement audit rules that alert on specific events, such as concurrent
    reading of multiple Secrets by a single user
 
-#### Additional ServiceAccount annotations for Secret management
+#### Additional ServiceAccount annotations for Secret management (deprecated)
```

Comment:
We can use a singular here.
```suggestion
#### ServiceAccount annotation for Secret management (deprecated) {#additional-serviceaccount-annotations-for-secret-management}
```

Reply:
The point of having a customized anchor ID is to make it easy to reference this section or subsection. In that spirit, I'd suggest we use a shorter ID which is
unique and clear enough in this page. For example, `#additional-annotation`.